crates/holochain/src/core/queue_consumer.rs

```diff
@@ -655,9 +655,12 @@ impl TriggerReceiver {
 }
 
 /// Create a future that will be ok with either a recv or a lagged.
-async fn rx_fut(rx: &mut broadcast::Receiver<()>) -> Result<(), QueueTriggerClosedError> {
+async fn rx_fut(rx: &mut broadcast::Receiver<String>) -> Result<(), QueueTriggerClosedError> {
```

Comment:
If these are never dynamic, should you use static refs?

```rust
const INIT: &'static &'static str = &"init";
async fn rx_fut(rx: &mut broadcast::Receiver<&'static &'static str>) -> Result<(), QueueTriggerClosedError> {
```
Note, the double reference is intentional, because it's [half the size memory-wise](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=b6fe4b70e45279ebc40006a83bdec885).

Reply:
Good idea. And, interesting. This is not a high frequency channel, so seems like a micro optimization. Is it clear that halving the payload of an already super small payload significantly outweighs the cost of dereferencing? I'll do it here but I'm wondering about using this technique more often.

Reply:
the double reference is totally a micro optimization, but also not that hard to do if you've already got a `&'static`... but maybe it diminishes code clarity, so kindof a wash?